exchange/exchange_test.go

```diff
@@ -2374,26 +2374,33 @@ func extractResponseTimes(t *testing.T, context string, bid *openrtb2.BidRespons
 	}
 }
 
-func newExchangeForTests(t *testing.T, filename string, expectations map[string]*bidderSpec, aliases map[string]string, privacyConfig config.Privacy, bidIDGenerator BidIDGenerator, hostSChainFlag, floorsFlag bool, hostBidValidation config.Validations, server exchangeServer) Exchange {
-	bidderAdapters := make(map[openrtb_ext.BidderName]AdaptedBidder, len(expectations))
-	bidderInfos := make(config.BidderInfos, len(expectations))
+func newExchangeForTests(t *testing.T, filename string, aliases map[string]string, privacyConfig config.Privacy, bidIDGenerator BidIDGenerator, exSpec *exchangeSpec) Exchange {
+	bidderAdapters := make(map[openrtb_ext.BidderName]AdaptedBidder, len(exSpec.OutgoingRequests))
+	bidderInfos := make(config.BidderInfos, len(exSpec.OutgoingRequests))
 	for _, bidderName := range openrtb_ext.CoreBidderNames() {
-		if spec, ok := expectations[string(bidderName)]; ok {
+		if spec, ok := exSpec.OutgoingRequests[string(bidderName)]; ok {
 			bidderAdapters[bidderName] = &validatingBidder{
 				t:             t,
 				fileName:      filename,
 				bidderName:    string(bidderName),
 				expectations:  map[string]*bidderRequest{string(bidderName): spec.ExpectedRequest},
 				mockResponses: map[string]bidderResponse{string(bidderName): spec.MockResponse},
 			}
-			bidderInfos[string(bidderName)] = config.BidderInfo{ModifyingVastXmlAllowed: spec.ModifyingVastXmlAllowed}
+			ortbVersion, found := exSpec.ORTBVersion[string(bidderName)]
+			if !found {
+				ortbVersion = "2.5"
```

Comment:
why not leave it blank if the string is not found? The vast majority of bidders today do not have a version value.

Reply:
Agree it would be preferable to leave it blank so it more accurately reflects actual configuration state.

Reply:
Just wanted to be more explicit. it will work both ways. Removed.